content/en/about/case-studies/bluecore/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Bluecore Leverages Istio to Migrate to Kubernetes"
+linkTitle: "Bluecore Leverages Istio to Migrate to Kubernetes"
+quote: "Istio enables an approach that allows you to start building right out of the box." 
+author:
+    name: "Shray Kumar"
+    image: "/img/authors/shray-kumar.jpg"
+companyName: "Bluecore"
+companyURL: "https://bluecore.com/"
+logo: "/logos/bluecore.png"
+skip_toc: true
+skip_byline: true
+skip_pagenav: true
+doc_type: article
+sidebar_force: sidebar_case_study
+type: case-studies
+weight: 30
+---
+
+Bluecore is a multichannel personalization platform that specializes in highly personalized emails, onsite messages, and paid media ads delivered at scale. Since Bluecore's unique offering requires processing customer recommendation data as well as sending a large quantity of personalized digital communications, they deal with ingesting, processing, and sending tons of data. it's a huge workload; during a recent Black Friday, they sent over 2 billion emails.
+
+![2 billion emails and 12,000 Kubernetes pods](./bluecore-statistics.png)
+
+Along with volume, Bluecore needs to be speedy. They have service agreements with most of their partners guaranteeing that any customer will receive an email, onsite message, or ad within a few hours of a job being submitted, which means that processing speed is of vital importance. They accomplish this feat using both a monolithic application running on App Engine, and a Google Kubernetes Engine (GKE) cluster running about 12,000 pods.
+
+## The Challenge: Monolithic architecture and increasing data traffic
+
+Bluecore is a large operation with serious requirements around data transfer and processing. They were facing a daunting challenge; the amount of data they need to process continually increases. They knew their pipelines would get overloaded if their architecture couldn't be updated to handle increasing demands.
+
+An examination of their architecture revealed that the monolithic application was going to be the biggest challenge. The Bluecore development team realized that to enable future growth, it was time to begin migrating to a more flexible and scalable infrastructure.
+
+Kubernetes offered a path to scale, and since Bluecore was already running on Google's App Engine, it seemed like migrating portions of their workflow to GKE would be a no-brainer. However, the majority of Bluecore's engineers didn't have sufficient experience with containerized applications. This could make migrating to GKE a painful transition.
```

Comment:
```suggestion
Kubernetes offered a path to scale, and since Bluecore was already running on App Engine, it seemed like migrating portions of their workflow to GKE would be a no-brainer. However, the majority of Bluecore's engineers didn't have sufficient experience with containerized applications. This could make migrating to GKE a painful transition.
```